LetsDoStuff.WebApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using LetsDoStuff.Domain;
+using LetsDoStuff.Domain.Models;
+using LetsDoStuff.WebApi.SettingsForAuth;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+
+namespace LetsDoStuff.WebApi.Controllers
+{
+    [Route("api/account")]
+    public class AccountController : Controller
+    {
+        private LdsContext context;
+
+        public AccountController(LdsContext context)
+        {
+            this.context = context;
+        }
+
+        [HttpGet]
+        public string GetTestOutput()
+        {
+            return "test output";
+        }
+
+        [HttpPost("token")]
+        public IActionResult Token(string userLogin, string userPasseword)
```

Comment:
It's completely unsafe. Authentication data should be sent out in the body.

Reply:
Yes, and let's call the method meaningful. Seems like we have a verb for that process from the very dawn of internet - login)

Reply:
When I was coding I didn't think about all of it. I was focused only on making a simple auth. And I made this as simple as possible for improving this after we choose the better way. After all, I'm not good enough at website development. So it's nice, that you guys have already given me advice.